Server/Services/RevolutionaryDiscordBotService.cs

```diff
@@ -772,12 +794,30 @@ private async Task<Image<Rgb24>> GenerateSlowDaysSinceImage(WatchedKeyword keywo
         return daysSinceImage;
     }
 
-    private SlashCommandBuilder BuildDsucCommand()
+    private SlashCommandBuilder BuildDaysSinceCommand()
     {
         return new SlashCommandBuilder()
-            .WithName("dsuc")
+            .WithName("dayssince")
             .WithDescription(
-                "Displays how many days have passed since \"underWater Civilizations\" were last brought up.");
+                "Displays how many days have passed since a keyword was last brought up")
+            .AddOption(new SlashCommandOptionBuilder()
+                .WithName("keyword")
+                .WithRequired(true)
+                .WithDescription("Sets what keyword to display")
+                .AddChoice("UnderwaterCivs", "underWaterCiv")
+                .AddChoice("SentientPlants", "sentientPlant")
+                .WithType(ApplicationCommandOptionType.String));
+    }
+
+    private int GetAllKeywordMentions()
+    {
+        int mentions = 0;
+        foreach (WatchedKeyword keyword in database.WatchedKeywords)
```

Comment:
This is inefficient, you should use `database.WatchedKeywords.Sum` so that the sum is calculated on the database and not all data is loaded here into C# objects.

Reply:
Fixed